examples/load/README.md

```diff
@@ -0,0 +1,155 @@
+# File based task/template/handler definition
+
+This proposal introduces directory based task, template, and handler definition.
+
+## Configuration
+
+```
+[load]
+ enabled = true
+ dir="/path/to/directory"
+```
+
+`dir` specifies the directory where the definition files exist.
+
+The specified directory will check to see if any of three subdirectories `tasks`, `templates`, and `handlers` exists.
```

Comment:
The configured `dir` may contain any of these sub-directories: `tasks`, `templates`, and `handlers`.